ui/src/style/components/table-graph.scss

```diff
@@ -67,6 +67,18 @@
     background-color: $g6-smoke;
     color: $g16-pearl;
   }
+  &__isFieldName:hover {
+    cursor: pointer;
+  }
+  &__isFieldName:before {
+    font-family: 'icomoon';
+    content: "\e902";
+    font-size: 17px;
+  }
+  &__isSortedUp {
```

Comment:
eventually these need something in them.

Reply:
yes they do.

Reply:
What uppppp

Reply:
hahah.. yes allex?? we can do this whenever you are not hanging out with Iris